### PR TITLE
add orm and pg

### PR DESCRIPTION
**Pull request by VitaliiKostyrka**

### CODE REVIEW

File: api/db/configdb.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/postgres"
+)
+
+var db *gorm.DB
+
+func init() {
+	connection, e := gorm.Open("postgres", "user=quiz password=quiz dbname=quiz")
+	if e != nil {
+		fmt.Print(e)
```
Comment (sonishc) [inline api/db/configdb.go]: add row
`panic("Could not connect to database")`

File: api/db/configdb.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package db
+
+import (
+	"../models"
+	"fmt"
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/postgres"
```
Comment (sonishc) [inline api/db/configdb.go]: Is it nessesary?

File: api/db/question.go [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package db
+
+import (
+	"../models"
```
Comment (sonishc) [inline api/db/question.go]: spaces

File: api/db/configdb.go (hunk 2 above)
Comment (VitaliiKostyrka) [reply api/db/configdb.go]: Yes, you need Postgres dialects to connect to PostgresSQL

**Pull request merged**